lib.sh

```diff
@@ -1307,7 +1307,7 @@ fi
 docker_update_specific() {
 if does_this_docker_exist "$1" "$2"
 then
-    docker run --rm --name watchtower -v /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower --cleanup --run-once "$1"
+    docker run --rm --name temporary-watchtower -v /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower --cleanup --run-once "$1"
```

Comment:
I am not sure if hyphens are allowed for docker container names?
We can also use a different one then this. it just has to be something else than `watchtower`